internal/config/blockio/blockio.go

```diff
@@ -0,0 +1,59 @@
+package blockio
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"sigs.k8s.io/yaml"
+
+	"github.com/intel/goresctrl/pkg/blockio"
```

Comment:
The very same commit that adds the use of a new package should also have changes to go.mod adding it. Otherwise you break git-bisect.

Reply:
Thanks @kolyshkin!

I reordered/squashed commits, and made sure that cri-o builds without errors on every step.